settings/browser.qml

```diff
@@ -38,8 +40,12 @@ Page {
                 title: qsTrId("settings_browser-ph-browser")
             }
 
+            DisabledByMdmBanner {
+                active: !AccessPolicy.browserEnabled
+            }
```

Comment:
Are there any needs to protect the settings? If not, I'd just skip these parts.

Reply:
Just consistency with the camera blocking really.

Reply:
But does it make sense really?

Reply:
I can imagine someone filing a bug if it's still active.

Reply:
Let's keep it there.